From: Egbert Eich <eich@suse.de>
Date: Wed May 17 10:46:11 2017 +0200
Subject: Avoid a date string in compiled code.
Patch-mainline: never
Git-commit: bb0446da315743fe3b6e393641accf8747ee0923
References: 

A DATE string in a binary will cause the package to be marked
updated whenever it changes.

Signed-off-by: Egbert Eich <eich@suse.com>
---
 ompi/debuggers/ompi_msgq_dll.c | 4 ++++
 1 file changed, 4 insertions(+)
diff --git a/ompi/debuggers/ompi_msgq_dll.c b/ompi/debuggers/ompi_msgq_dll.c
index a89e5f1..350b0e0 100644
--- a/ompi/debuggers/ompi_msgq_dll.c
+++ b/ompi/debuggers/ompi_msgq_dll.c
@@ -194,7 +194,11 @@ static char mqs_version_str[OMPI_MAX_VER_SIZE];
 char *mqs_version_string (void)
 {
     return "Open MPI message queue support for parallel"
+#ifndef OPAL_PACKAGE_STRING
            " debuggers compiled on " __DATE__;
+#else
+           " debuggers compiled for " OPAL_PACKAGE_STRING;
+#endif
     int offset;
     offset = snprintf(mqs_version_str, OMPI_MAX_VER_SIZE-1,  
                       "Open MPI message queue support for parallel debuggers ");
